crates/astria-bridge-withdrawer/src/withdrawer/mod.rs

```diff
@@ -81,7 +105,9 @@ impl BridgeService {
         let Self {
             shutdown_token,
             api_server,
+            submitter,
             ethereum_watcher,
+            state: _state,
```

Comment:
add todo for connecting this to the api server

Reply:
[it already is](https://github.com/astriaorg/astria/blob/3b3199218f0a065889a5b0b126cec2db0837d0e5/crates/astria-bridge-withdrawer/src/withdrawer/mod.rs#L91) - should it live inside submitter or watcher instead?